docker/README.md

```diff
@@ -0,0 +1,11 @@
+This directory contains a Dockerfile for building the
+`uniffi-ci` docker image that we use for running tests
+in CI. Build a new version of this docker image, use:
+
+```
+docker build -t rfkelly/uniffi-ci -f Dockerfile-build .
+docker push rfkelly/uniffi-ci
+```
+
+That only works if you're `rfkelly`; We need to figure out
+a better strategy for maintainership of said docker image.
```

Comment:
FYI I've already pushed a new version of this Docker image using the changed below (and muddling through that process inspired me to write this short doc).